### PR TITLE
Add Get Started Checklist to Editor Page

### LINKED ISSUE #45: Add a get started page for editors
@jlpalomino will work on a friendlier get started page for editors! 

* Editor template should be easy to find on that page!! 
* Should have a list of steps
* Maybe some discuss of changing the labels for the issues
* pyopensci badges - when that should go on a users repo
* Version number -- should the final version of the software have a _dev index on it.

### PR DESCRIPTION
**Pull request by jlpalomino**
This PR modifies the page for Guidelines for Editors to add a Get Started Checklist. This PR also splits out the text for the editor in chief (from review editor guidelines) onto a new page. 

Fixes #45

### CODE REVIEW

**Comment by choldgraf** [comment]
this looks awesome! ping me when it's ready to go :-)

File: submissions/editor_guide.md [hunk 1]
```diff
@@ -54,7 +48,7 @@ When these sources of information are not enough:
 * look for users of the package or of the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter).
 * You can also search for authors/maintainers of related packages on [PyPI](https://pypi.org/search/).
```
Comment (lwasser) [inline submissions/editor_guide.md:49]: we can ask on pyopensci twitter (i've had great success with this approach so far!! )

**Comment by lwasser** [comment]
@jlpalomino i did some significant renaming of directories and such so you may need to update from master ... this is looking great!!

**Comment by lwasser** [comment]
merging this to support our new editor. @jlpalomino will work on things more next week and we can review more closely then!!

**Comment by lwasser** [comment]
closing given #69

**Pull request merged**